OracleTuxedo/OracleTSAM/dockerfiles/12.2.2/Dockerfile

```diff
@@ -100,10 +100,8 @@ RUN mkdir /etc/envs && \
       bash -c "echo . /etc/envs/bashrc >> /etc/bashrc" && \
       bash -c "echo . /etc/envs/bashrc >> /u01/oracle/.bashrc" && \
       cd /u01/oracle && \
-      jar xf /u01/$ORACLT_PKG1 && \
-      jar xf /u01/$ORACLT_PKG2 && \
-      chmod -R +x instantclient_* && \
-      mv /u01/oracle/instantclient* /u01/oracle/instantclient && \
+      rpm -i --nodeps /u01/$ORACLT_PKG1 /u01/$ORACLT_PKG2 && \
```

Comment:
Please do not use the `rpm` utility to install packages. This shuold be replaced by `yum localinstall` so that package dependencies are resolved and installed as well.

Installing with `--nodeps` is very dangerous and should never be done.